mimic/rest/maas_api.py

```diff
@@ -30,6 +30,16 @@
 Request.defaultContentType = 'application/json'
 
 
+def _find_index_of_id(xs, val):
```

Comment:
Gotcha. I think Ruby uses a similar method separator? I've definitely seen `#` used that way before, I just thought you'd like to know :).